ansible/files/telemetry.sh

```diff
@@ -48,6 +48,9 @@ fi
 PRESTO_CONFIG_SCRIPT=$(cat <<EOF
 while ! pgrep presto > /dev/null; do sleep 1; done
 
+# install presto plugins
+sudo -u presto aws s3 sync $TELEMETRY_CONF_BUCKET/plugins/ /usr/lib/presto/plugin/
```

Comment:
We could, but we try and not rely on external services (github, npm, dockerhub, etc) for dependencies and artifacts.